okapi/services/fieldnotes/upload/docs.xml

```diff
@@ -0,0 +1,49 @@
+<xml>
```

Comment:
Make sure that your method returns the `ocde-specific` infotag in [`services/apiref/method`](https://www.opencaching.de/okapi/services/apiref/method.html).